Main.java

```diff
@@ -1,5 +1,8 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("hello");
+        int[] numbers = {1, 2, 3};
+        for (int i = 0; i <= numbers.length; i++) {
```

Comment:
The condition 'i <= numbers.length' may cause an ArrayIndexOutOfBoundsException. Consider using 'i < numbers.length' to iterate within the bounds of the array.